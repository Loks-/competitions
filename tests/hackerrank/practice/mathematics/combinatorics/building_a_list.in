2
2
ab
3
xyz
