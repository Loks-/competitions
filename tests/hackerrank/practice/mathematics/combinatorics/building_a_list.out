a
ab
b
x
xy
xyz
xz
y
yz
z
